### PR TITLE
Change spec and protocol id

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
When we changed the spec name we invalidated the old chain specs. This cleans up the old vitalam stuff by:

1. Change the runtime spec to not be tied to this "node"
2. Remove all instances of vitalam
3. Update the protocol id to dscp rather than vam

Note we will have to re-generate the staging chainspec but I can't do that yet

### CODE REVIEW

File: node/src/command.rs [hunk 1]
```diff
@@ -50,8 +50,6 @@ impl SubstrateCli for Cli {
         Ok(match id {
             "dev" => Box::new(chain_spec::development_config()?),
             "" | "inteli-stage" => Box::new(chain_spec::inteli_stage_config()?),
```
Comment (n3op2) [inline node/src/command.rs:52]: maybe we should call this `inteli-staging`

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -88,215 +88,10 @@ pub fn development_config() -> Result<ChainSpec, String> {
     ))
 }
 
-pub fn vitalam_config() -> Result<ChainSpec, String> {
-    ChainSpec::from_json_bytes(&include_bytes!("../res/vitalam.json")[..])
-}
-
 pub fn inteli_stage_config() -> Result<ChainSpec, String> {
     ChainSpec::from_json_bytes(&include_bytes!("../res/inteli-stage.json")[..])
```
Comment (n3op2) [inline node/src/chain_spec.rs:92]: Have not seen `inteli-stage.json` in this PR

File: node/src/command.rs (hunk 1 above)
Comment (mattdean-digicatapult) [reply node/src/command.rs:52]: it's called `inteli-stage` in the infra hence the name

File: node/src/chain_spec.rs (hunk 2 above)
Comment (mattdean-digicatapult) [reply node/src/chain_spec.rs:92]: Indeed, I needed the feature branch to build to regenerate the genesis. That's done and pushed now

**Comment by mattdean-digicatapult** [comment]
FYI the chain spec for this was created using:

```
./scripts/make-new-cluster-genesis.sh -o oem:oem:1152921504606846976 -o t1:t1:1152921504606846976 -o t2:t2:1152921504606846976 -v oem-0:oem:oem -v oem-1:oem:oem -a bn:oem:oem -a ipfs-bn-light:oem:oem -a api-ipfs-light:oem:oem -a id-light:oem:oem -v t1-0:t1:t1 -v t1-1:t1:t1 -a api-ipfs-light:t1:t1 -a id-light:t1:t1 -a api-ipfs-light:t2:t2 -a id-light:t2:t2  -c ghcr.io/digicatapult/dscp-node:v3.0.1-chore_change-spec-protocol-id inteli-stage
```

from the https://github.com/digicatapult/dscp-flux-infra repo. The companion PR that updates the secrets is at https://github.com/digicatapult/dscp-flux-infra/pull/20

**Review by n3op2 — APPROVED** [review]

**Pull request merged**